### PR TITLE
[SchemaRegistry] update core + SR version

### PR DESCRIPTION
**Pull request by swathipil**
for issue #21460

### CODE REVIEW

File: sdk/schemaregistry/azure-schemaregistry/setup.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
         'msrest>=0.5.0',
-        'azure-core<2.0.0,>=1.19.0'
+        'azure-core<2.0.0,>=1.19.1'
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry/setup.py:72]: I think azure core is gonna GA the llc in v1.20 this week, so we could probably just bump the min dep to be 1.20?

Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry/setup.py:72]: ok, then I can wait to merge this until tomorrow (I think they said thursday)

**Review by yunhaoling — APPROVED** [review]

**Pull request merged**